modules/node/src/cfCore/cfCore.provider.ts

```diff
@@ -2,19 +2,29 @@ import { IMessagingService, MessagingServiceFactory } from "@connext/messaging";
 import { ConnextNodeStorePrefix } from "@connext/types";
 import { Provider } from "@nestjs/common";
 import { FactoryProvider } from "@nestjs/common/interfaces";
-import { Wallet } from "ethers";
-import { HDNode } from "ethers/utils";
+import { fromMnemonic } from "ethers/utils/hdnode";
 
+import { migrateToPatch1 } from "../cfCoreMigrations/patch1";
 import { ConfigService } from "../config/config.service";
-import { CFCoreProviderId, MessagingProviderId } from "../constants";
+import { CF_PATH, CFCoreProviderId, MessagingProviderId } from "../constants";
 import { LockService } from "../lock/lock.service";
 import { CLogger, freeBalanceAddressFromXpub } from "../util";
-import { CFCore, EXTENDED_PRIVATE_KEY_PATH } from "../util/cfCore";
+import { CFCore } from "../util/cfCore";
 
 import { CFCoreRecordRepository } from "./cfCore.repository";
 
 const logger = new CLogger("CFCoreProvider");
 
+// TODO: where should this live?
+const LATEST_CF_STORE_VERSION = 1;
```

Comment:
our latest version should live in a table, the latest version they support should live in the node. then we should make sure we note if migrations have been performed in our table as well

Reply:
Why does our latest version need to live in a table? We will have to change app code every time we want to migrate so it seems like it should be in the code itself. Also, they are building a more robust migration framework so we should probably wait for now.